recipes/libudev-devel-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: libudev-devel-cos6-x86_64
+  version: 147
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libudev-devel-147-2.73.el6_8.2.x86_64.rpm
+    sha256: 318322d5e63b27bf423e011e9b3d8863e5b1aa5ac57de131add0a85c8e6bbbfe
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/udev-147-2.73.el6_8.2.src.rpm
+    folder: source
```

Comment:
We need to either add a checksum or drop the source (I’ve gone for the latter).

Reply:
only for the devel packages though?